packages/components/app/styles/components/accordion.scss

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// ACCORDION
+//
+
+@use "../mixins/button" as *;
+@use "../mixins/focus-ring" as *;
+
+$accordion-item-padding: 16px;
+$accordion-item-medium-border-radius: 6px;
+$accordion-gap: 12px;
+
+// ACCORDION COMPONENT (wrapper)
+
+.hds-accordion {
+  display: flex;
+  flex-direction: column;
+  gap: $accordion-gap;
+}
+
+// ACCORDION ITEM COMPONENT (nested child)
+
+.hds-accordion-item {
+  background: var(--token-color-surface-primary);
+  border-radius: $accordion-item-medium-border-radius;
+
+  &.hds-accordion-item--does-not-contain-interactive {
+    box-shadow: var(--token-surface-mid-box-shadow);
+
+    &:hover,
+    &.mock-hover {
+      box-shadow: var(--token-surface-high-box-shadow);
+    }
+  }
+
+  &.hds-accordion-item--contains-interactive {
+    border: 1px solid var(--token-color-border-primary);
+  }
+}
+
+// TOGGLE BLOCK
+
+.hds-accordion-item__toggle {
+  position: relative;
+  display: flex;
+  gap: $accordion-gap;
+  align-items: center;
+  padding:
+    $accordion-item-padding
+    $accordion-item-padding
+    $accordion-item-padding
+    12px;
+}
+
+.hds-accordion-item__button {
+  padding: 0;
+
+  &:hover { cursor: pointer; }
+
+  .flight-icon-chevron-down { color: var(--token-color-foreground-primary); }
```

Comment:
I have mixed feelings about this here: on the `parent-contains-interactive` this color is already set by the secondary button mixin, so maybe we could think of moving this under the `hds-accordion-item__button--is-not-interactive` block as `color` property, and let the SVG just inherit it via `currentColor`?